packages/react-native/Libraries/AppDelegate/RCTUIConfiguratorProtocol.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <UIKit/UIKit.h>
+#import <React/RCTConvert.h>
+
+@class RCTRootView;
+
+@protocol RCTUIConfiguratorProtocol
+/**
+ * The default `RCTColorSpace` for the app. It defaults to `RCTColorSpaceSRGB`.
+ */
+- (RCTColorSpace)defaultColorSpace;
```

Comment:
I changed this from @property, It was read-only anyways so I don't think it needs to be a property